go/stats/counters.go

```diff
@@ -122,6 +185,159 @@ func (c *Counters) Counts() map[string]int64 {
 	return counts
 }
 
+// Help returns the help string.
+func (c *Counters) Help() string {
+	return c.help
+}
+
+// CountersWithLabels provides a labelName for the tagged values in Counters
+// It provides a Counts method which can be used for tracking rates.
+type CountersWithLabels struct {
```

Comment:
Technically, this is only a single label and not multiple labels.

i.e. it should be named `CountersWithLabel` ?

Reply:
Okay. Let's keep it as is for now.